brush-shell/tests/cases/builtins/kill.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Builtins: kill"
+cases:
+  - name: "kill -l"
+    ignore_stderr: true
+    # note that we dont use 'kill -l' because of the shape of the bash list
```

Comment:
Any other ideas on how to get some test coverage for `kill -l`?

I'm not sure how much we want to assume about the format of `bash`'s output, but something like this would extract a "flat" list from its implementation of `kill -l`:

```bash
siglist=($(kill -l | sed -e "s/[[:digit:]]\+)//g"))
for s in ${siglist[@]}; do echo $s; done
```

Reply:
@reubeno should we also hide EXIT from the printed list? The test is currently failing because of EXIT